### DISCUSSION TITLE
https://mnfienen.github.io/talulat/2023/08/23/day22.html?fbclid=IwAR0OV29-j4A_XQFZ0Cq44JW3mjPsozhCmo48vdmWp5N4ohCY4iav5BcIjRs_aem_ASIM1aW85kxLUoDST80R7M4kmlgR2O4KPLxq8AeG7FhqQ2koG725keNM9jITQKTAlTQ

### DISCUSSION THREAD

**Discussion by giscus** [mnfienen/talulat#31] [Comments]
# https://mnfienen.github.io/talulat/2023/08/23/day22.html?fbclid=IwAR0OV29-j4A_XQFZ0Cq44JW3mjPsozhCmo48vdmWp5N4ohCY4iav5BcIjRs_aem_ASIM1aW85kxLUoDST80R7M4kmlgR2O4KPLxq8AeG7FhqQ2koG725keNM9jITQKTAlTQ

Day 22 - Some long days bring out the grump, even when they are beautiful!

https://mnfienen.github.io/talulat/2023/08/23/day22.html?fbclid=IwAR0OV29-j4A_XQFZ0Cq44JW3mjPsozhCmo48vdmWp5N4ohCY4iav5BcIjRs_aem_ASIM1aW85kxLUoDST80R7M4kmlgR2O4KPLxq8AeG7FhqQ2koG725keNM9jITQKTAlTQ

**Comment by SmittysGitHub** [comment]
Congrats on making it to Nova Scotia!  I am impressed with only one grumpy day!
